stable/redis/templates/configmap.yaml

```diff
@@ -15,6 +15,10 @@ data:
 {{- end }}
   master.conf: |-
     dir {{ .Values.master.persistence.path }}
+{{- if .Values.master.configmap }}
+    # User-supplied master configuration:
+{{ .Values.master.configmap | indent 4 }}
```

Comment:
Should we use `toYaml`?

Reply:
It seems `toYaml` is useful if we are filling up the entire block. Since we are having some additional configs here it cannot be used.

To illustrate with an example, if I use `toYaml` is get the following. Note the `|-` that has be inserted which bungles up everything

```yaml
redis.conf: |-
    # User-supplied configuration:
    |-
      # Enable AOF https://redis.io/topics/persistence#append-only-file
      appendonly yes
      # Disable RDB persistence, AOF persistence already enabled.
      save ""
```

Reply:
👍 fair enough